modular_citadel/code/modules/client/preference_setup/loadout/loadout_general_cit.dm

```diff
@@ -0,0 +1,29 @@
+// General loadout for Citadel additions.
+
+/datum/gear/tennis_ball
```

Comment:
Is it even avoids putting in donator items, automatically? Either way, i wanted to exclude rainbow one as well because of squeaky noises (and don't let me remind you of current *weh situation), though i would be glad if there was vote whenever all tennis balls would be squeaky, only one (rainbow one) and it's available from loadout or only one admin-spawnable.

Reply:
It would be easier to use a geartweak, actually. You'd need one main entry and then just the geartweak code I posted.